source/install/ee-prod-rhel-7.rst

```diff
@@ -107,7 +130,7 @@ Set up Mattermost Server
 2. Download `any version of Mattermost Enterprise Edition <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing:
 
    -  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
-   -  Where ``vX.X.X`` is typically the latest Mattermost release version, which is currently ``v3.3.0``.
+   -  Where ``vX.X.X`` is typically the latest Mattermost release version, which you can find out from the `download page <https://about.mattermost.com/download/>`_.
```

Comment:
@JeffSchering made a small tweak here to reference the download page to find the latest version number rather than hard-coding it.